### PR TITLE
release/public-v2: addd RRFs v1alpha regression tests

### PR DESCRIPTION
**Pull request by climbfuji**
## Description

- add regression tests for new suite FV3_RRFS_v1alpha
- remove redundant `effr_in=.true.` from namelist templates
- change `NEW_BASELINE` in rt.sh to avoid conflicts when creating new baselines for release/public-v2 and develop at the same time

## Testing

Regression tests were run on the following platforms as for the previous release/public-v2 PRs: cheyenne.{gnu,intel}, hera.{gnu,intel}, jet.intel, gaea.intel. Logs updated in the PR.

## Dependencies

https://github.com/NCAR/ccpp-physics/pull/518
https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/pull/61
https://github.com/NOAA-EMC/fv3atm/pull/197
https://github.com/ufs-community/ufs-weather-model/pull/276

### CODE REVIEW

**Review by DusanJovic-NOAA — APPROVED** [review]

**Review by junwang-noaa — APPROVED** [review]

**Review by mkavulich — APPROVED** [review]
Looks good to me. Once it is merged we will update the SRW app to point to the latest hash so we can use the new suite in the release branch.

**Comment by climbfuji** [comment]
> Looks good to me. Once it is merged we will update the SRW app to point to the latest hash so we can use the new suite in the release branch.

These PRs are waiting for the approval and merge of the GFDL_atmos_cubed_sphere PR https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/pull/61 - everything is ready to go.

**Comment by junwang-noaa** [comment]
Dom, who would you recommend to review the dycore PR#61?

**Comment by climbfuji** [comment]
> Dom, who would you recommend to review the dycore PR#61?
> […](#)

Anyone from GFDL will be fine. I asked Rusty to review, but so far no reply. Note that this PR is for the NOAA-GFDL fork, not the NOAA-EMC fork (because it is for the public release branch) - can you assign reviewers and merge the code there?

**Comment by junwang-noaa** [comment]
No, I can't. Only GFDL code managers can do that.

**Comment by climbfuji** [comment]
I updated the submodule pointer for fv3atm and reverted the change to `.gitmodules`. Since the PR has been approved by the code managers, I can merge it myself - will do so.

**Pull request merged**
